consensus/mint-client/src/config.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Command line configuration for the consensus mint client.
+
+use clap::Parser;
+use mc_crypto_keys::{DistinguishedEncoding, Ed25519Private};
+use mc_util_uri::ConsensusClientUri;
+use std::fs;
+
+#[derive(Debug, Parser)]
+#[clap(
+    name = "mc-consensus-mint-client",
+    about = "MobileCoin Consensus Mint Client"
+)]
+pub struct Config {
+    /// URI of consensus node to connect to.
+    #[clap(long, env = "MC_CONSENSUS_URI")]
+    pub node: ConsensusClientUri,
+
+    /// The key to sign the transaction with.
+    #[clap(long, parse(try_from_str=load_key_from_pem), env = "MC_PRIVATE_KEY")]
```

Comment:
Nit: Spaces around equal sign.
```suggestion
    #[clap(long, parse(try_from_str = load_key_from_pem), env = "MC_PRIVATE_KEY")]
```